<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatternLoadWindow</class>
 <widget class="QDialog" name="PatternLoadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Point of view configuration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Configuration of points of view</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="nbPatternsLabel">
           <property name="text">
            <string>Number of patterns:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="nbPatternsInfoLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="nbFeaturesLabel">
           <property name="text">
            <string>Number of features (columns):</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="nbFeaturesInfoLabel">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="nbPointsOfViewLabel">
           <property name="text">
            <string>Number of points of view:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="nbPoVSpinBox"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="startLabel">
           <property name="text">
            <string>Start:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QSpinBox" name="povStartSpinBox"/>
         </item>
         <item row="1" column="3">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="3">
          <widget class="QPushButton" name="saveButton">
           <property name="text">
            <string>Save changes</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QComboBox" name="povComboBox"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="endLabel">
           <property name="text">
            <string>End:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QSpinBox" name="povEndSpinBox"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="patternTable">
        <property name="minimumSize">
         <size>
          <width>557</width>
          <height>300</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>557</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusReportLabel">
     <property name="text">
      <string>Ready.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PatternLoadWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PatternLoadWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
